src/script/conversation/ConversationCellState.ts

```diff
@@ -377,17 +380,35 @@ const _getStateUserName = {
   },
 };
 
+const _getStateVerified = {
+  description: (conversationEntity: Conversation) => {
+    const isMLSProtocol = conversationEntity.protocol === ConversationProtocol.MLS;
+
+    return t(isMLSProtocol ? 'tooltipConversationAllE2EIVerifiedShort' : 'tooltipConversationAllDevicesVerified');
+  },
+  icon: () => ConversationStatusIcon.NONE,
+  match: (conversationEntity: Conversation) => {
+    const lastMessageEntity = conversationEntity.getNewestMessage();
+    const isE2EIVerificationMessage = lastMessageEntity?.type === ClientEvent.CONVERSATION.E2EI_VERIFICATION;
+    const {systemMessages: unreadSystemMessages} = conversationEntity.unreadState();
+
+    return isE2EIVerificationMessage && unreadSystemMessages.length > 0;
+  },
+};
+
 export const generateCellState = (
   conversationEntity: Conversation,
 ): {description: string; icon: ConversationStatusIcon | void} => {
   const states = [
+    _getStateVerified,
```

Comment:
I think we might have a bug there. 
Can you try:
- sending a few messages in a conversation (you should see the message counter + the subtext with the content of the message)
- sending a reply
- sending a @ mention
- then the verification message

-> I believe the `reply + message counter` will disapear because the system message takes precedence. 

Can you confirm?